c2cgeoportal/scaffolds/update/CONST_alembic/main/versions/72eaad5daf34_add_field_ogc_server_version_wms.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017, Camptocamp SA
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+"""Add field ogcserver.version_wms
+
+Revision ID: 72eaad5daf34
+Revises: 809650bd04c3
+Create Date: 2018-05-23 13:19:13.802243
+"""
+
+from alembic import op, context
+from sqlalchemy import Column
+from sqlalchemy.types import Unicode
+
+# revision identifiers, used by Alembic.
+revision = '72eaad5daf34'
+down_revision = '809650bd04c3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    schema = context.get_context().config.get_main_option('schema')
+
+    op.add_column('ogc_server', Column('version_wms', Unicode), schema=schema)
```

Comment:
Default to 1.1.1?

Reply:
This is done in Entry class `version_wms or '1.1.1'`

Reply:
For me the default value should be visible in the admin interface...

Reply:
I also like the idea that we leave the choice to the app if the user does not want to force.
(=> set this only if you know what you are doing)
What about if you want to change the default value in future version ?

Reply:
In 2.3 we can add a description for the field (default to 1.1.1)

Reply:
OK for the description :-)